4.0/start.sh

```diff
@@ -33,13 +46,48 @@ if [ ! -d ${TOKENIZER_DIR} ]; then
   # More reading: https://github.com/mediagis/nominatim-docker/pull/274/
   echo "No tokenizer configuration found. Copying from persistent volume into project directory."
   cp -r /var/lib/postgresql/12/main/tokenizer ${TOKENIZER_DIR}
-  chown -R nominatim:nominatim ${PROJECT_DIR}
 fi
 
+chown -R nominatim:nominatim ${PROJECT_DIR}
+
 service postgresql start
 
 cd ${PROJECT_DIR} && sudo -u nominatim nominatim refresh --website
 
+# start replication on container (re)start
+if [ "$REPLICATION_URL" != "" ]; then
+  mkdir -p $(dirname ${NOMINATIM_REPLICATION_LOG:-/var/log/nominatim/replication.log})
+  sudo -E -u nominatim nominatim replication > ${NOMINATIM_REPLICATION_LOG:-/var/log/nominatim/replication.log} 2>&1 &
+  cat << EOT > /etc/logrotate.d/nominatim_replication
+${NOMINATIM_REPLICATION_LOG:-/var/log/nominatim/replication.log} {
```

Comment:
How are you executing the logrotate? Cron or systemd don't run in the container.

See: https://github.com/mediagis/nominatim-docker/blob/master/4.0/Dockerfile#L10

Reply:
Didn't see this, thanks